.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
+*.d text eol=lf
+*.bmp binary
```

Comment:
Why we care about bmp again?
Also not sure why force line ending, pretty sure people have their configurations set for them. (I guess this makes sense for dual boot setups though)
</bikeshed>

Reply:
We have file format support for bmp, but otherwise we don't care about it.

I'm adding this file because a previous attempt at this PR I switched it to \r\n and Manu wasn't all that happy about it (I typically don't bother setting line endings). So to guarantee line endings are consistent no matter who contributes, we have this file.

Reply:
I 'complained' because you commit windows line endings to the repo.
This isn't the right solution... This shouldn't be forced, you should just have autocrlf set correctly for your computer in your local client.
Windows clients need autocrlf in general, but that's a local setting, not a repo seeing.